sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EpidemiologicalDataCaseSteps.java

```diff
@@ -51,6 +53,26 @@ public EpidemiologicalDataCaseSteps(
       EnvironmentManager environmentManager) {
     this.webDriverHelpers = webDriverHelpers;
 
+    When(
+        "I set Start and End of activity by current date in Activity as Case form for DE version",
+        () -> {
+          webDriverHelpers.fillInWebElement(
+              START_OF_EXPOSURE_INPUT, DATE_FORMATTER_DE.format(LocalDate.now()));
+          webDriverHelpers.fillInWebElement(
+              END_OF_EXPOSURE_INPUT, DATE_FORMATTER_DE.format(LocalDate.now()));
+        });
+    When(
+        "I fill Description field in Activity as Case form",
+        () -> {
+          webDriverHelpers.fillInWebElement(ACTIVITY_DESCRIPTION, faker.book().title());
+        });
+
+    When(
+        "I check that edit Activity as Case vision button is visible and clickable",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(OPEN_SAVED_ACTIVITY_BUTTON);
```

Comment:
but why do we need it as a separated method?

Reply:
This separation method make us sure that new case is added and appeared in Activity as Case table as a record